### PR TITLE
change to lib

### PR DESCRIPTION
**Pull request by StrixLuca**
test yes

### CODE REVIEW

**Comment by mafewtm** [comment]
If you're updating the locales to ox_lib locales, you need to transfer over the other locales as well. We don't want to lose them all

**Comment by StrixLuca** [comment]
ah oke thanks for the info

**Review by DevX32 — CHANGES_REQUESTED** [review]
better mark plate as variable and use qbx.getvehicleplate

**Comment by DevX32** [comment]
and convert the closet (player and vehicle) @StrixLuca

File: fxmanifest.lua [hunk 1]
```diff
@@ -4,15 +4,15 @@ game 'gta5'
...
+ox_lib 'locale'
+
 
 shared_scripts {
-    '@ox_lib/init.lua',
-    '@qbx_core/modules/utils.lua',
-    '@qbx_core/shared/locale.lua',
-    'locales/en.lua',
-    'locales/*.lua',
+     '@ox_lib/init.lua',
```
Comment (BerkieBb) [inline fxmanifest.lua]: indentation is off here or under this

File: fxmanifest.lua [hunk 2]
```diff
@@ -4,15 +4,15 @@ game 'gta5'
...
+    '@qbx_core/modules/lib.lua',
 }
 
+
```
Comment (BerkieBb) [inline fxmanifest.lua]: this can go

**Review by solareon — APPROVED** [review]

**Review by mafewtm — APPROVED** [review]

**Pull request merged**